tests/queries/0_stateless/03370_join_identifiers.sql

```diff
@@ -0,0 +1,28 @@
+DROP TABLE IF EXISTS t0;
+DROP TABLE IF EXISTS t1;
+
+CREATE TABLE t0
+(
+    `id` INT UNSIGNED NOT NULL,
+    `rev` INT UNSIGNED NOT NULL,
+    `content` varchar(200) NOT NULL
+)
+ENGINE = MergeTree
+PRIMARY KEY (id, rev);
+
+CREATE TABLE t1
+(
+    `id` INT UNSIGNED NOT NULL,
+    `rev` INT UNSIGNED NOT NULL,
+    `content` varchar(200) NOT NULL
+)
+ENGINE = MergeTree
+PRIMARY KEY (id, rev);
+
+SELECT SUM(t1.rev) AS aggr
+FROM t1
+INNER JOIN t0 AS right_0 ON t1.id = right_0.id
+INNER JOIN t1 AS right_1 ON t1.id = right_1.id;
```

Comment:
Understand that the test check query analysis, but it'd be good to have some data in the tables

Reply:
Added